sdk/storage/azure-storage-queue/azure/storage/queue/_shared/authentication.py

```diff
@@ -38,15 +38,7 @@ def _wrap_exception(ex, desired_type):
     msg = ""
     if ex.args:
         msg = ex.args[0]
-    if sys.version_info >= (3,):
```

Comment:
One lint error in this file now that you've removed the use of `sys`.
`azure/storage/queue/_shared/authentication.py:8: [W0611(unused-import), ] Unused import sys`

Reply:
Hello @jalauzon-msft ! Thank you so much for the review! I've removed the import statement for `sys` in a new commit. Please let me know if I can improve on the PR. 

I love a good challenge, so modernizing the entire storage blob will be a treat 😆 
In addition to that, I want to PR some more technical issues, but don't know where to start really. May you please refer me to some beginner-friendly issues? Thanks and have a wonderful day!

Reply:
Hmm, I'm not entirely sure of any issues that we have that would be beginner-friendly but let me look through the backlog and get back to you. Thanks.